src/midcom/datamanager/template/form.php

```diff
@@ -259,7 +259,7 @@ public function choice_widget_expanded(FormView $view, array $data)
         $string = '<fieldset ' . $this->renderer->block($view, 'widget_container_attributes') . '>';
         foreach ($view as $child) {
             $string .= $this->renderer->widget($child);
-            $string .= $this->renderer->label($child);
+            $string .= $this->renderer->humanize($this->renderer->label($child));
```

Comment:
shouldn't `humanize` be called in the `label` function? Because here, the return value already includes HTML markup